bip-taro-proof-file.mediawiki

```diff
@@ -127,13 +130,16 @@ where:
 * <code>anchor_tx</code>: is the transaction spending the <code>previous_outpoint</code>. This transaction commits to at least a single Taro asset tree within one of its outputs.
 * <code>taro_taproot_proof</code>: is a nested TLV that can be used to prove either inclusion or a taro asset, or the lack of a taro commitment via the <code>taro_commitment_exclusion_proof</code>.
 * <code>taproot_exclusion_proofs</code>: is a series of _exclusion_ proofs that prove that the other outputs in a transaction don't commit to a valid taro asset. This re-uses the <code>taro_taproot_proof</code> structure, but will only contain an <code>taro_commitment_exclusion_proof</code> value and not also a <code>taro_taproot_proof</code> value.
+* <code>split_root_proof</code>: is an optional <code>taro_taproot_proof</code> that proves the inclusion of the split commitment's root asset in case of an asset split.
+* <code>taro_input_splits</code>: is an optional list of nested full proofs for any additional inputs found within the resulting asset.
 
 The final flat proof file has the following format:
-* <code>file_sum</code>: 32-bytes - 256 sum over the remaining contents of the file
-* <code>file_version</code>: 4-bytes - version of proof file
-* <code>taro_proofs</code>:<code>BigSize</code> length prefixed serialized proofs
-
-// TODO(roasbeef): make the file_sum instead an MMR over the set? allows for probabilistic verification of file, prove certain transition fragments
+* [<code>u32</code>:<code>file_version</code>] version of proof file
```

Comment:
Would be good to also update the proof verification pseudocode to reflect the per-proof checksum verification?

Reply:
Ah yes, good catch! I'm not very good with Python-like pseudo code, so I hope it's understandable what I wrote.